data/fields/min_height.json

```diff
@@ -0,0 +1,9 @@
+{
+    "key": "min_height",
+    "minValue": 0,
+    "type": "number",
+    "label": "Height of bottom (Meters)",
```

Comment:
This should be in Title Case like other field labels.

Reply:
fixed in 4e60539d328c444bf37b94e96e189be99565b261, thanks for noticing!